build-monitor-plugin/src/test/java/com/smartcodeltd/jenkinsci/plugin/assetbundler/PathToAssetTest.java

```diff
@@ -11,12 +11,12 @@
 import static org.junit.Assert.assertThat;
 
 public class PathToAssetTest {
-
+    private static final String INDEX_LESS_PATH = "less/index.less";
```

Comment:
Maybe instead of extracting a constant we could extract a factory method producing a `PathToAsset`, taking `base` as its parameter?